infor-design/enterprise-wc-examples
# 316/931 - Add IdsToolbar examples to Angular project

**What's in this PR?**
As part of infor-design/enterprise-wc#931 I needed to test the IdsToolbar in an Angular environment.  We didn't have them yet so I've added a few in this PR mostly just to satisfy testing requirements

**Related issues:**
- Related to infor-design/enterprise-wc#316
- Closes infor-design/enterprise-wc#931

**How to test:**
- Pull/build/run
- Open http://localhost:4200/ids-toolbar and review examples (some still have illegal constructor errors)
- Make sure link to Toolbar examples show up on http://localhost:4200
